taxi/tests/test_views.py

```diff
@@ -0,0 +1,129 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from django.urls import reverse
+from taxi.models import Manufacturer, Car, Driver
+
+MANUFACTURER_LIST_URL = reverse("taxi:manufacturer-list")
+CAR_LIST_URL = reverse("taxi:car-list")
+CAR_DETAIL_URL = reverse("taxi:car-detail", kwargs={"pk": 1})
+DRIVER_LIST_URL = reverse("taxi:driver-list")
+DRIVER_DETAIL_URL = reverse("taxi:driver-detail", kwargs={"pk": 1})
+DRIVER_CREATE_URL = reverse("taxi:driver-create")
+
+
+class PublicManufacturerViewsTests(TestCase):
+
+    def test_login_required_manufacturer_list(self):
+        response = self.client.get(MANUFACTURER_LIST_URL)
+        self.assertNotEqual(response.status_code, 200)
+
+
+class PublicCarViewsTests(TestCase):
+
+    def test_login_required_car_list(self):
+        response = self.client.get(CAR_LIST_URL)
+        self.assertNotEqual(response.status_code, 200)
+
+    def test_login_required_car_detail(self):
+        response = self.client.get(CAR_DETAIL_URL)
+        self.assertNotEqual(response.status_code, 200)
+
+
+class PublicDriverViewsTests(TestCase):
+
+    def test_login_required_driver_list(self):
+        response = self.client.get(DRIVER_LIST_URL)
+        self.assertNotEqual(response.status_code, 200)
+
+    def test_login_required_driver_detail(self):
+        response = self.client.get(DRIVER_DETAIL_URL)
+        self.assertNotEqual(response.status_code, 200)
+
+
+class PrivatManufacturerViewsTests(TestCase):
```

Comment:
The class name `PrivatManufacturerViewsTests` should be `PrivateManufacturerViewsTests` to maintain consistency and correct spelling.